test/ios_test_runner_unit_test.sh

```diff
@@ -433,6 +433,25 @@ function test_ios_unit_simulator_id() {
   expect_log "Executed 3 tests, with 0 failures"
 }
 
+function test_ios_unit_test_dot_separated_command_line_args() {
+  create_sim_runners
+  create_ios_unit_tests
+  do_ios_test //ios:PassingUnitTest \
+    --test_arg="--command_line_args=arg1,arg2,arg3" || fail "should pass"
+
+  expect_log "Test Suite 'PassingUnitTest' passed"
+}
+
+function test_ios_unit_test_multiple_command_line_args() {
+  create_sim_runners
+  create_ios_unit_tests
+  do_ios_test //ios:PassingUnitTest \
+    --test_arg="--command_line_args=arg1" \
+    --test_arg="--command_line_args=arg2" || fail "should pass"
+
+  expect_log "Test Suite 'PassingUnitTest' passed"
```

Comment:
the other tests here actually verify these were passed, can you do that? i think you can check in NSUserDefaults to get these to see them

Reply:
@keith 
Thanks for the comment. I've added tests to check if the arguments have been passed to the app runtime using ProcessInfo.
`bazel test //test:ios_test_runner_unit_test` still doesn't pass because other test fails. I'll work on that after this PR.

Reply:
everything is passing on CI it looks like, so im not sure which one you mean?

Reply:
This test is tagged "manual" so it's disabled on CI.
https://github.com/bazelbuild/rules_apple/blob/d15beb478c05e493a87620af0ee67ab0ece41632/test/BUILD#L407-L420

Reply:
hrm we should probably enable now and toss 32 bit support